fiaas_deploy_daemon/config.py

```diff
@@ -278,6 +278,7 @@ def _parse_args(self, args):
         tls_parser.add_argument("--tls-certificate-issuer-type-overrides", help="Issuers to use for specified domain suffixes",
                                 default=[],
                                 action="append", type=KeyValue, dest="tls_certificate_issuer_type_overrides")
+        tls_parser.add_argument("--tls-certificate-ready", help="Check whether certificates are ready", default=False)
```

Comment:
Instead of allowing any kind of value, add an `action="store_true"` like the options of enable/disable (for example `disable-deprecated-managed-env-vars`)